pkgs/development/tools/backblaze-b2/default.nix

```
@@ -20,6 +20,10 @@ python3Packages.buildPythonApplication rec {
 
   checkInputs = with python3Packages; [ pytestCheckHook ];
 
+  preBuild = ''
+    export SETUPTOOLS_SCM_PRETEND_VERSION="${version}"
+  '';
```

looks a little cleaner, IMO
```suggestion
    SETUPTOOLS_SCM_PRETEND_VERSION= version;
```